### PR TITLE
Sasha pasha branch

### PR DESCRIPTION
**Pull request by AlexSyt**

### CODE REVIEW

**Comment by polarbearing** [comment]
Hi @AlexSyt - good to see all the progress here. I am testing nightly builds, so it would be even more helpful if there were some hints about what features are being implemented, and what bugs are being fixed. So I could keep my eyes in target when testing them.

**Comment by AlexSyt** [comment]
Hi, @polarbearing.
Thank you for your attention to our work.
We are engaged in improvement and redesign of Map markers. But at the moment the development is carried out in a separate branch, so the latest changes are not yet available in the nightly builds.

File: OsmAnd/src/net/osmand/plus/mapmarkers/MapMarkersDbHelper.java [hunk 1]
```diff
@@ -0,0 +1,511 @@
...
+		return conn;
+	}
+
+	private void onCreate(SQLiteConnection db) {
+		db.execSQL(MARKERS_TABLE_CREATE);
+		db.execSQL(GROUPS_TABLE_CREATE);
+		saveExistingMarkersToDb();
+	}
+
+	private void onUpgrade(SQLiteConnection db, int oldVersion, int newVersion) {
```
Comment (crimean) [inline OsmAnd/src/net/osmand/plus/mapmarkers/MapMarkersDbHelper.java]: So after upgrade we drop all existing markers?

Comment (AlexSyt) [reply OsmAnd/src/net/osmand/plus/mapmarkers/MapMarkersDbHelper.java]: This is an option for the development time.

**Pull request merged**